lib/Bio/EnsEMBL/DataCheck/Checks/DisplayLabels.pm

```diff
@@ -0,0 +1,67 @@
+=head1 LICENSE
+
+Copyright [2018-2020] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the 'License');
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an 'AS IS' BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package Bio::EnsEMBL::DataCheck::Checks::DisplayLabels;
```

Comment:
I think this name is too generic; and most people would assume it's about xrefs, but it's not, it's about prediction transcripts which have a column with the same name as in the xref table. I suggest 'PredictionTranscriptLabels'.